packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -1635,8 +1642,8 @@ class DiagnosticsProperty<T> extends DiagnosticsNode {
 }
 
 /// [DiagnosticsNode] for an instance of [Diagnosticable].
-class _DiagnosticableNode<T extends Diagnosticable> extends DiagnosticsNode {
```

Comment:
Now that this is public it probably requires more documentation explaining what it's for and so forth.
The current docs violate our style guide (in that they're something you could write even if you didn't know anything about DiagnosticableNode).

Reply:
documented all methods without an @override